CMakeLists.txt

```diff
@@ -13,7 +13,9 @@ if (NOT CMAKE_BUILD_TYPE)
 endif()
 
 # Redirecting the default installation path /usr/local to /usr no need to use -DCMAKE_INSTALL_PREFIX =/usr
-set(CMAKE_INSTALL_PREFIX /usr)
+if (CMAKE_INSTALL_PREFIX STREQUAL "/usr/local")
+	set(CMAKE_INSTALL_PREFIX "/usr")
```

Comment:
Little fix:
```suggestion
    set(CMAKE_INSTALL_PREFIX "/usr")
```

Reply:
So, it is no more possible, to install to `/usr/local/`?

Reply:
If we accept the fixes offered by OPNA2608, then there is no point in this option and it can be removed.

Reply:
> So, it is no more possible, to install to `/usr/local/`?

Fixed

Reply:
@Masterkatze, thank you!